Network/Network.cpp

```diff
@@ -598,14 +600,18 @@ namespace WPEFramework
             params["interface"] = m_netUtils.getInterfaceDescription(interface);
             if (ipv6Addr != "")
             {
+#ifdef NET_NO_LINK_LOCAL_ANNOUNCE
                 if  (!m_netUtils.isIPV6LinkLocal((const std::string) ipv6Addr))
+#endif
```

Comment:
When "NET_NO_LINK_LOCAL_ANNOUNCE" is not defined, this will result in a dangling "else" and thus a compilation error.

Reply:
Corrected